hugo/content/showcase/statemachine.html

```diff
@@ -5,12 +5,38 @@
 layout: showcase-page
 url: "/showcase/statemachine"
 img: "/assets/Langium_Statemachine.svg"
+file: "scripts/statemachine.tsx"
 description: A language that captures the functionality of a state machine. Demonstrated by modeling a traffic light.
 geekdochidden: true
 draft: false
 noMain: true
 ---
 
+<style>
```

Comment:
Please use Tailwind css classes instead

Reply:
I'm inclined to agree as well. When possible using the [tailwindcss utility classes](https://tailwindcss.com/docs/utility-first) that are provided is also a pretty good bet. It makes the styling very portable (i.e. the classes can go anywhere and still achieve the same effect). However if you run into a few special cases where you can't quite find the correct utility class, I would recommend updating the stylesheet in the tailwinds folder.

Although I have also definitely written inline stylesheets for other parts of the showcase, but I probably should have been using tailwind as well in those contexts.